### PR TITLE
[Relax][PyTorch] Fix batch normalization training mode correctness

### PR DESCRIPTION
**Pull request by guan404ming**
## Why

Batch normalization in training mode would threw away updated statistics.

## How

batch_norm(...) → keep all 3 elements, pad to 5 for PyTorch

### CODE REVIEW

**Comment by guan404ming** [comment]
cc @mshr-h @tlopex
